runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -102,8 +102,9 @@ pub fn measure_actions(
 
 // TODO: super-ugly, can achieve the same via higher-level wrappers over POSIX read().
 #[inline(always)]
+#[allow(clippy::missing_safety_doc)]
 pub unsafe fn syscall3(mut n: usize, a1: usize, a2: usize, a3: usize) -> usize {
-    asm!("syscall"
+    llvm_asm!("syscall"
```

Comment:
Guess https://github.com/rust-lang/rfcs/pull/2843 hints that this is just the new syntax.

Reply:
```
error: use of deprecated item 'asm': the syntax of asm! will change soon, use llvm_asm! to avoid breakage
   --> runtime/runtime-params-estimator/src/testbed_runners.rs:107:5
    |
107 |     asm!("syscall"
    |     ^^^ help: replace the use of the deprecated item: `llvm_asm`
    |
```
I don't know anything about this `asm` macro, but just applied to suggested change. Though I see it failed to compile with CI, so I'll take a closer look at it.

Reply:
I bumped the version of rust nightly I am using locally to get around a bug in RLS that is fixed in a later version (https://github.com/rust-lang/rls-vscode/issues/749), and this is why I was seeing this deprecation error. The version of nightly the project is currently on does not yet have `llvm_asm`. I'll revert this change for now and we can bring it back in if we bump the rust version for the whole project.

By the way, what are others using for code navigation? I assume I must be the only one using RLS in VSCode if there is an issue with it in the nightly version the project is currently on.

Reply:
Many people use CLion, we can get you a license.
I think @nearmax was making sure VSCode also works though.

Reply:
I stopped using VSCode because I was causing too much pain..